Template/README.md

```diff
@@ -0,0 +1,31 @@
+## Entidades disponíveis
+
+# Post (Rede social)
+- Title
+- Text
+- Message
+- Image
+- Attachments (Coleção de anexos, podendo ser Imagem, Vídeo, Localização...)
+
+# Message (TBR)
+- Title
+- Text
+- Image
+
+# Item (TBR)
+- Title
+- Text
+- Image
+- Price
+
+## Objetos disponíveis
+- Model
+- View
+- Utils
```

Comment:
se for com métodos, esse não vai rolar...
O webview só aceita um objeto com métodos C# mapeados....
Agora se for para adicionar utils javascript eu colocaria em uma biblioteca js e disponibilizaria aqui nos libraries

Reply:
Teria que ser C# para ter acesso aos recursos do app, por exemplo: nosso GetDate (que já retorna do servidor ou equipamento)... pegar endereço do servidor (para consumir outro serviço/api)...
Uma solução seria o objeto que vai para o WebView ser só um wrapper de outros objetos (Fragmento, classe BO, classe Utils), que no cabeçalho do html setaria em cada variável...

Reply:
pois é, à principio os objetos que a gente passa depois, não tem métodos mapeados pro C#
Tipo.. o objeto principal que hoje retorna o model, transforma o model em um json pra poder ser usado em javascript.
Logo, json não aceita métodos...
Ainda que aceitasse, como dizer que essas "strings" são relativas à um método C#?
por isso comentei... só o objeto principal aceita métodos mapeados do C# e só podemos ter um objeto javascript com essa característica.

Reply:
Fica para descobrirmos uma forma...
Caso realmente não seja possível pelo WebView, daria para ser um lib JS que se comunica com o app... uma api local, mensagens...